client/my-sites/site-settings/form-discussion.jsx

```diff
@@ -475,7 +475,7 @@ class SiteSettingsFormDiscussion extends Component {
 		} = this.props;
 		return (
 			<FormFieldset className="site-settings__moderation-settings">
-				<FormLegend>{ translate( 'Comment Moderation' ) }</FormLegend>
+				<FormLegend>{ translate( 'Comment moderation' ) }</FormLegend>
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.